Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11.4-slim
+LABEL mantainer="volodymyr_dolgyi@ukr.net"

```

Comment:
There is a typo in the `LABEL` instruction. It should be `maintainer` instead of `mantainer`.